VideoWebRTC/app/src/main/java/com/shivam/androidwebrtc/tutorial/CompleteActivity.java

```diff
@@ -125,13 +125,13 @@ private void connectToSignallingServer() {
         try {
             // For me this was "http://192.168.1.220:3000";
             // $ hostname -I
-            String URL = "https://calm-badlands-59575.herokuapp.com/";
+            String URL = "http://10.173.1.175:3030/";// "https://calm-badlands-59575.herokuapp.com/"; //
```

Comment:
```suggestion
            String URL = "https://calm-badlands-59575.herokuapp.com/";
```
Maybe the better option would be to throw a fatal uncaught error here